src/sql/workbench/common/customInputConverter.ts

```diff
@@ -57,11 +57,7 @@ export function convertEditorInput(input: EditorInput, options: IQueryEditorOpti
 		//Notebook
 		uri = getNotebookEditorUri(input, instantiationService);
 		if (uri) {
-			let fileName: string = 'untitled';
-			if (input) {
```

Comment:
We're already inside an if(input) check so no need to do this conditional check. It has no value.